frontend/src/games/aim-trainer-game.tsx

```diff
@@ -66,13 +74,31 @@ const AimTrainerGame = () => {
   const [difficulty, setDifficulty] = useState(Difficulty.EASY);
   const [isLoading, setIsLoading] = useState(false);
   const [dotsLeft, setDotsLeft] = useState(0);
+  const dotsLeftRef = useRef(0);
   const [score, setScore] = useState(0);
+  const scoreRef = useRef(0);
+  const [username, setUsername] = useState('');
+  const [highscores, setHighscores] = useState([] as Highscore[]);
+  const [loadingHighscores, setLoadingHighscores] = useState(false);
 
   const dots: MutableRefObject<Array<HTMLDivElement>> = useRef([]);
 
+  async function fetchHighscores() {
+    setLoadingHighscores(true);
+    const tmp = await fetch(`${BACKEND_URL}/aimtrainergame/highscores?=${NUMBER_OF_HIGHSCORES}`);
+    const highscores = (await tmp.json()) as Highscore[];
```

Comment:
Sitos eilutes gali throwint, tai tada gal geriau nekeist highscore lenteles?

Reply:
Nzn ar fronte labai verta tokius dalykus ziureti, pagal tai nevertins musu. Aisku tikrai gera praktika, bet cia produktas kuri mes rodysim vertintojui tik tikrai pamate kad ant musu kompiuteriu pilnai veikia

Reply:
Jo, fair, sita irgi rasiau labiau just in case.